benchmark/libsinsp/utils.cpp

```diff
@@ -28,3 +28,47 @@ static void BM_sinsp_split(benchmark::State& state)
 	}
 }
 BENCHMARK(BM_sinsp_split)->Repetitions(2);
+
+static void BM_sinsp_concatenate_paths_relative_path2(benchmark::State& state)
+{
+	std::string path1 = "/tmp/";
+	std::string path2 = "foo/bar";
+	for(auto _ : state)
+	{
+		sinsp_utils::concatenate_paths(path1, path2);
+	}
+}
+BENCHMARK(BM_sinsp_concatenate_paths_relative_path2)->Repetitions(2);
```

Comment:
if we want some reliable data in CI we probably need to bump the number of repetitions to at least `20`. It will require more time but it will provide something more meaningful. BTW since we probably want to repeat all the tests at least `20` times we could use `--benchmark_repetitions` directly from the command line instead of bumping it here, WDYT?

Reply:
Agree with the CLI argument! I'll add it in a moment

Reply:
I'd also remove the `->Repetitions(2)` from the benchmarks declaration, since it enforces `2` as number of repetitions (it is not just a default, it can't be overridden from `--benchmark_repetitions` CLI arg too!)

Reply:
Done!